tests/acceptance/let-helper-test.js

```diff
@@ -59,4 +61,21 @@ describe('Acceptance: let helper', function() {
       });
     });
   });
+
+  if (emberVersionIs('greaterThan', "2.0.0")) {
```

Comment:
I'm a tiny bit skeptical of adding a dep for this. But I can be sold on it very easily :p